doc/release_notes_1.0.rst

```diff
@@ -0,0 +1,606 @@
+.. _release_notes_1.0:
+
+ACRN v1.0 (May 2019)
+####################
+
+We are pleased to announce the release of Project ACRN version 1.0,
+which is ACRN major milestone focused on automotive.
+ACRN is a flexible, lightweight reference hypervisor, built with 
+real-time and safety-criticality in mind, optimized to streamline embedded 
+development through an open source platform. Check out the :ref:`introduction` for more information.
+All project ACRN source code is maintained in the https://github.com/projectacrn/acrn-hypervisor 
+repository and includes folders for the ACRN hypervisor, the ACRN device 
+model, tools, and documentation. You can either download this source code as
+a zip or tar.gz file (see the `ACRN v1.0 GitHub release page
+<https://github.com/projectacrn/acrn-hypervisor/releases/tag/v1.0>`_)
+or use Git clone and checkout commands::
+
+   git clone https://github.com/projectacrn/acrn-hypervisor
+   cd acrn-hypervisor
+   git checkout v1.0
+
+The project’s online technical documentation is also tagged to correspond 
+with a specific release: generated v1.0 documents can be found at https://projectacrn.github.io/1.0/. 
+Documentation for the latest (master) branch is found at https://projectacrn.github.io/latest/.
+ACRN v1.0 requires Clear Linux OS version 29070 or newer. Please follow the 
+instructions in the :ref:`getting-started-apl-nuc`.
+
+Version 1.0 new features
```

Comment:
These aren't all "new" features since the last release, so this whould say something like ``Version 1.0 major features``.